index.d.ts

```diff
@@ -129,11 +129,11 @@ export interface AssertContext {
 	 * Assert that function throws an error or promise rejects.
 	 * @param error Can be a constructor, regex, error message or validation function.
 	 */
-	throws(value: (() => void) | Promise<{}>, error?: ErrorValidator, message?: string);
+	throws(value: (() => void) | Promise<{}>, error?: ErrorValidator, message?: string): void;
```

Comment:
`throws` return the error thrown though: https://github.com/sindresorhus/ava#throwsfunctionpromise-error-message